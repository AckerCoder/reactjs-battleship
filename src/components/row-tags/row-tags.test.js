import React from "react";
import { shallow } from "enzyme";
import RowTags from "./row-tags.component";

it("Expect to Match Snapshot", () => {
  expect(shallow(<RowTags />)).toMatchSnapshot();
});

it("Expect to render CustomInput Component", () => {
  expect(shallow(<RowTags />).length).toEqual(1);
});
