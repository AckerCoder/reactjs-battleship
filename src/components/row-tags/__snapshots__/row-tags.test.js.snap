// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to Match Snapshot 1`] = `
<div
  className="row-tags"
>
  <div
    className="row-tag"
  >
    A
  </div>
  <div
    className="row-tag"
  >
    B
  </div>
  <div
    className="row-tag"
  >
    C
  </div>
  <div
    className="row-tag"
  >
    D
  </div>
  <div
    className="row-tag"
  >
    E
  </div>
  <div
    className="row-tag"
  >
    F
  </div>
  <div
    className="row-tag"
  >
    G
  </div>
  <div
    className="row-tag"
  >
    H
  </div>
  <div
    className="row-tag"
  >
    I
  </div>
  <div
    className="row-tag"
  >
    J
  </div>
</div>
`;
